src/js/util.js

```diff
@@ -23,12 +23,26 @@ const showStep = (step, stepList) => {
     return
   }
 
+  const stepperNode = closest(step, Selectors.STEPPER)
+
   const activeStep = stepList.filter(step => step.classList.contains(ClassName.ACTIVE))
   if (activeStep.length) {
     activeStep[0].classList.remove(ClassName.ACTIVE)
   }
+  for (let stepItem in stepList) {
+    stepList[stepItem].querySelector(Selectors.LINK).removeAttribute('aria-current')
```

Comment:
if you query the link inside the `stepItem` store it in a variable please, and you have to query that line another time, two lines below 👍 

```js
const button = stepList[stepItem].querySelector(Selectors.LINK)